server/lib/utils.js

```diff
@@ -1,3 +1,33 @@
+const moment = require('moment');
```

Comment:
All of this was here before - I simplified `summarizeLedger` a bit because it did more processing than needed here.